chapters/01-introduction.Rmd

```diff
@@ -2,11 +2,19 @@
+In many fields of science and in physics especially, scientists need to estimate the probability density function (PDF) from which a set of data is drawn. If the underlying mechanisms are well understood a so called parametric approach can be used. One can describe the model mathematically and fit it to the experimental data by some goodness-of-fit criterion like log-likelihood or $\chi^2$. 
 
-To combat this so called non-parametric methods like histograms are used. By summing the the data up in discrete bins we can approximate the underlying parent distribution, without needing any knowledge of the physical interactions. However there are also many more sophisticated non-parametric methods, one in particular is Kernel Density Estimation (KDE), which can be looked at as a sort of generalized histogram [@rosenblatt1956remarks]. 
+Oftentimes however the underlying mechanisms are too complex to be fully understood analytically and the parametric methods fail. The knowledge of the system is to poor to describe the model as mathematical function.
 
-Histograms tend to produce PDFs that are highly dependent on bin width and bin positioning, meaning the interpretion of the data changes by a lot by two arbitrary parameters. KDE circumvents this problem by replacing each data point with a so called kernel that specifies how much it influences its neighbouring regions as well. The kernels themselves are centered at the data point directly, eliminating the need for arbitrary bin positioning[@duong2001]. Since KDE still depends on kernel width (instead of bin width), one might argue that this is not a major improvement. However, upon closer inspection, one finds that the underlying PDF does depend less strongly on the kernel width than histograms on bin width and it is much easier to specify rules for an approximately optimal kernel width than it is to do so for bin width[@lerner2013]. In addition, by specifying a smooth kernel, one gets a smooth distribution as well, which is often desirable or even expected from theory.
+In the particle accelerator at CERN for instance, they record a whooping 25 gigabytes of data per second[@cern], resulting from the process of many physical interactions that occur almost simultaneously. It is not feasible to anticipate features of the distribution one observes experimentally, as it is comprised of many different distributions, which result from all the different physical interactions.
+
+To combat this, so called non-parametric methods are used, which do not need a predefined mathematical model. 
+
+The perhaps simplest non-parametric method is the Histogramm. By summing the the data up in discrete bins the underlying parent distribution can be approximated, without needing any knowledge of underlying processes.
+
+However there are also many more sophisticated non-parametric methods, one in particular is Kernel Density Estimation (KDE), which can be looked at as a sort of generalized histogram [@rosenblatt1956remarks].
```

Comment:
[4] change with line 17: first talk about the drawbacks, then mention the improved versions